lib/manager/components/deployment/PeliasPanel.js

```diff
@@ -106,7 +106,7 @@ class PeliasPanel extends Component<Props, State> {
 
     const file = files[0]
     // TODO: more extensive csv validation?
-    if (file.type !== 'text/csv') {
+    if (file.type !== 'text/csv' && file.type !== 'application/vnd.ms-excel') {
```

Comment:
What MIME type would be returned on a Windows machine that doesn't have MS Excel?

Reply:
From the research I've done, it looks like this happens even if Office isn't installed. I couldn't find any bug reports or similar featuring any other mime types.